generated/java/build.gradle

```diff
@@ -124,14 +122,14 @@ project(':generated:java:google-cloud-logging-v2') {
 
 project(':generated:java:google-cloud-pubsub-v1') {
   dependencies {
-    compile 'com.google.api.grpc:grpc-google-cloud-pubsub-v1:' + generated_pubsub_version
+    compile 'com.google.api.grpc:grpc-google-cloud-pubsub-v1:' + generated_grpc_version
   }
 }
 
 project(':generated:java:google-cloud-language-v1') {
```

Comment:
We need separate `project` declarations for the different versions of an API, because they are in different folders. I think this is better in a separate PR.